src/test/ui/parser/item-free-const-no-body-semantic-fail.rs

```diff
@@ -3,5 +3,4 @@
 fn main() {}
 
 const A: u8; //~ ERROR free constant item without body
-const B; //~ ERROR free constant item without body
```

Comment:
Was this `//~ ERROR` marker intentionally removed? I still see the error in the stderr.